scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -470,7 +498,7 @@ task M2 {
         --max-mnp-distance 0
   >>>
   runtime {
-      docker: "us.gcr.io/broad-gatk/gatk:4.1.1.0"
+      docker: select_first([gatk_docker_override, "us.gcr.io/broad-gatk/gatk:4.1.1.0"])
```

Comment:
I'm not sure what the solution here is, but does keeping this as the default version mean that we'll have to keep updating it when there are new releases? That's the way it already was, so it's not necessarily a problem, I just want to remember that once this PR is merged and released we should go back and update this default.